lib/matplotlib/axes/_axes.py

```diff
@@ -1091,55 +1091,70 @@ def eventplot(self, positions, orientation='horizontal', lineoffsets=1,
                   linelengths=1, linewidths=None, colors=None,
                   linestyles='solid', **kwargs):
         """
-        Plot identical parallel lines at specific positions.
-
-        Plot parallel lines at the given positions.  positions should be a 1D
-        or 2D array-like object, with each row corresponding to a row or column
-        of lines.
+        Plot identical parallel lines at the given positions. positions should
```

Comment:
Keep a single summary line.
Mark arguments as `*such*`.  (See "documenting matplotlib" entry in the faq.)